src/scripts/components/Wallet.js

```diff
@@ -29,26 +29,32 @@ class Wallet extends Component<Props> {
   render () {
     const balance = this.props.user.balances.PTI
     return (
-      <Card title="Wallet">
+      <Card
+        title="Wallet"
+        maxWidth
+        footer={
+          ACTIVATE_SECURE_WALLET &&
+          !this.props.isWalletSecured && (
+            <TextButton
+              data-test-id="secure-wallet"
+              onClick={this.secureWallet}
+              accent
+            >
+              Secure Wallet
+            </TextButton>
+          )
+        }
+      >
         <Text>
           Your address is{' '}
-          <strong data-test-id="user-address">{this.props.userAddress}</strong>
+          <Strong primary data-test-id="user-address">
+            {this.props.userAddress}
+          </Strong>
         </Text>
         <Text>
-          You have <strong>{paratii.eth.web3.utils.fromWei(balance)}</strong>{' '}
-          PTI
+          You have{' '}
```

Comment:
Translations!

Reply:
Fine to leave for another PR

Reply:
Commented in https://github.com/Paratii-Video/paratii-portal/pull/769#discussion_r201203109